core/domain/stats_jobs_continuous.py

```diff
@@ -238,12 +269,19 @@ class InteractionAnswerSummariesAggregator(
     """
     @classmethod
     def get_event_types_listened_to(cls):
+        """Returns a list of submitted answer events.
+
+        Returns:
+            list. A list of submitted answer events.
```

Comment:
Returns a list of event types that this class subscribes to.

Return:
    list(str). A list of submitted answer event type.

Reply:
Done!